### PR TITLE
[PRTL-2641] save custom interval bins

### PR DESCRIPTION
**Pull request by samrichca**
## Ticket Number

PRTL-2640
PRTL-2641

## Environment to be used in testing

- [ ] `prod`
- [x] `dev-oicr`
- [ ] `qa` (which version?)

## Description of Changes

Save custom interval bins when the user clicks the submit button in the continuous binning modal.

Also includes "and up" bin name change from PRTL-2640.

### CODE REVIEW

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js [hunk 1]
```diff
@@ -1509,11 +1509,12 @@ export default compose(
       },
     })
   ),
-  withProps(({ data, fieldName, variable }) => {
+  withProps(({ data: { explore }, fieldName, variable }) => {
```
Comment (samrichca) [inline src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: fix issue with `data.explore.cases` crashing the app when `explore` isn't available

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js [hunk 2]
```diff
@@ -926,20 +926,20 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
...
                     width: BOX_PLOT_RATIO,
                   }}
-                  >
+                       >
                     <span style={{
                       color: theme.greyScale3,
                       fontSize: '1.2rem',
                     }}
-                    >
+                          >
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: For the record, some of those closing tags realigning to the first prop would look less weird if the prop goes onto a new line. ☝️

Comment (samrichca) [reply src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: My auto formatting is consistently inconsistent, I'm going to look into it next week

Comment (caravinci) [reply src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: hm it's something at eslint I'm fixing on my commit. Don't sweat it ;) I gotchu!

**Pull request merged**